outputs.tf

```diff
@@ -14,7 +14,32 @@
  * limitations under the License.
  */
 
-output "buckets_list" {
-  value       = "${compact(split(" ", data.external.bucket_retrieval.result["buckets_list"]))}"
-  description = "The buckets list within the created project"
+output "forseti-server-vm-name" {
+  description = "Forseti Server role vm name"
+  value = "${google_compute_instance.forseti-server.name}"
+}
+
+output "forseti-server-vm-ip" {
+  description = "Forseti Server role ip address"
+  value = "${google_compute_instance.forseti-server.network_interface.address}"
+}
+
+output "forseti-client-vm-name" {
+  description = "Forseti Client role vm name"
+  value = "${google_compute_instance.forseti-client.name}"
+}
+
+output "forseti-client-vm-ip" {
+  description = "Forseti Cleint role vm name"
+  value = "${google_compute_instance.forseti-client.network_interface.address}"
+}
+
+output "foseti-server-service-account" {
+  description = "Forseti Server role service account"
+  value = "${google_service_account.forseti_server.email}"
+}
+
+output "foseti-client-service-account" {
+  description = "Forseti Client role service account"
+  value = "${google_service_account.forseti_client.email}"
 }
```

Comment:
One additional output request - can we get the name of the storage bucket? I was getting ready to dig through the bucket that we created to see what's there, but we don't expose that so I couldn't use Terraform to get that info.

(Other outputs look good though. Thanks!)

Reply:
Added and strung through the examples